### ISSUE TITLE
Other players can delete each other

### ISSUE THREAD

**Issue by adamland** [jameshd/playfooty#30]
I would delete all the rubbish players every time.

Perhaps this is why we never manage to get a game anymore because all I do is moan at the poor players when they give the ball away.

**Comment by jameshd** [comment]
you mean just let you delete yourself and the manager can delete everyone?

**Comment by adamland** [comment]
Yeah, the players can only delete themselves.  Obviously the Manager can delete everyone, he has the power like Prince Adam of Eternia
